similarity-search/src/main/java/org/phenotips/similarity/internal/SolrSimilarPatientsFinder.java

```diff
@@ -204,4 +214,36 @@ private long count(SolrQuery query)
         }
         return 0;
     }
+
+    private Collection<String> getPresentPhenotypeTerms(Patient patient)
+    {
+        Collection<String> termIds = new ArrayList<String>();
```

Comment:
Looks good, +1. It was not obvious from looking at https://github.com/phenotips/patient-network/pull/98#event-468759305. (BTW, in general, if merge is not urgent maybe we should wait until the author does the change just so that everyone is aware/can comment more/etc)